ui/src/constants/TutorialSections.tsx

```diff
@@ -66,6 +66,7 @@ export const TutorialSections: ITutorialSection[] = [
         shortInfo: 'broadcastShortInfo',
         title: 'broadcastTitle',
         icon: <CellTowerOutlined />,
+        linesOfCode: 8,
```

Comment:
Might need a comment somewhere on why this is here, and how it relates to the snippets generated by the backend.